script/glam/run_glam_sql

```diff
@@ -100,25 +104,62 @@ function run_view {
 }
 
 
-function run_desktop_sql {
-    local prod_project="moz-fx-data-shared-prod"
-    local tables_to_copy=(
-        "latest_versions"
-        "clients_histogram_aggregates_new_v1"
-    )
-
-    # TODO - make these computable rather than copying.
-    for table in "${tables_to_copy[@]}"; do
-        echo "Copying table $table"
-        bq cp --no_clobber "${prod_project}:${PROD_DATASET}.$table" "${PROJECT}:${DATASET}.$table";
+function run_partitioned_query {
+    local NUM_PARTITIONS=5
+    local NUM_SAMPLE_IDS=100
+    local PARTITION_SIZE=$((NUM_SAMPLE_IDS / NUM_PARTITIONS))
+    local table_name=$1
+    local partitioned_result=${2:-true}
+
+    run_query $table_name true 0 19
+    for partition in $(seq 1 $((NUM_PARTITIONS - 1))); do
+        local min_param=$((partition * PARTITION_SIZE))
+        local max_param=$((min_param + PARTITION_SIZE - 1))
+        run_query $table_name $partitioned_result $min_param $max_param \
+            $table_name "--append_table --noreplace"
```

Comment:
Isn't `--noreplace` only for `bq load`?  I don't think it does anything for `bq query`